.coveralls.yml

```diff
@@ -0,0 +1,2 @@
+service_name: travis-pro
+repo_token: ZUTKvNcViYlweUGjtm22HmNsjN2CX5Pqf
```

Comment:
Apparently public repos don't need repo_token. I think we can delete this file. 
https://github.com/lemurheavy/coveralls-public/issues/208

Reply:
Oh alright! Will remove now.

Reply:
Done.